src/Connection.php

```diff
@@ -281,26 +303,4 @@ protected function getDefaultSchemaGrammar()
     {
         return new Schema\Grammar();
     }
-
-    /**
-     * Set database.
-     *
-     * @param \MongoDB\Database $db
-     */
-    public function setDatabase(\MongoDB\Database $db)
```

Comment:
Wouldn't removal of this public method constitute a BC break?

Reply:
I double checked. This is unused function because `$db` property is local and only place where we set value to this is `__construct` method of same class